x-pack/legacy/plugins/siem/public/components/url_state/__snapshots__/index.test.tsx.snap

```diff
@@ -353,7 +353,7 @@ exports[`UrlStateContainer mounts and renders 1`] = `
                         }
                       }
                     >
-                      <withRouter(Connect(UrlStateContainer))
+                      <withRouter(Connect(Component))
```

Comment:
FYI https://github.com/adriantoine/enzyme-to-json/issues/137

Reply:
Thanks, I just read through it. I don't think it's worth doing the workaround to get the name back, if you're ok with that as sooner or later they will fix the bug.